index.js

```diff
@@ -5,3 +5,4 @@ const Server = require('./server');
 const config = require('./config');
 
 module.exports = new Server(config);
+module.exports.Server = Server;
```

Comment:
Do you think this is needed? In cases I need the flexibility to instantiate an instance of server, I do `const MockYeahServer = require('mockyeah/Server')`.

Reply:
@ryanricard It's more intuitive for me, but up to you.

Reply:
Is there a common practice you have observed with other libraries?

Reply:
👍 This isn't worth pressing.

Reply:
@ryanricard Yes, I've seen this pattern elsewhere in the Node ecosystem. I don't have specific examples. But if there's no need to worry about tree shaking for a client-side lib, exposing as a named export of the root module prevents the client from relying on importing from a specific file path (which must be supported indefinitely for backwards compatibility).